docs/ADD_OHLC_IMPLEMENTATION.md

```
@@ -0,0 +1,503 @@
+```python
+# When exporting to CSV, include OHLC data
+if chart_ohlc_df is not None and not chart_ohlc_df.empty:
+    chart_ohlc_df = chart_ohlc_df.copy()
+    chart_ohlc_df["type"] = "ohlc"
+    # Include in combined_df
+```
+
+### 5. CSV Format Changes
+
+**Current CSV columns (lines):**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)